packages/dev/core/src/scene.ts

```diff
@@ -4177,7 +4177,7 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
             const rtt = camera.outputRenderTarget;
             if (rtt.onClearObservable.hasObservers()) {
                 rtt.onClearObservable.notifyObservers(this._engine);
-            } else if (!rtt.skipInitialClear) {
+            } else if (!rtt.skipInitialClear && !camera.isRightCamera) {

```

Comment:
just wonder if we could not for skipInitialClear on rightCameraRTT (assuming probabylon wrongly it is different from the left one :-)) so that we do not need the additional check ?

Reply:
I'll look into improving that

Reply:
discussed offline